pcsx2/CDVD/CDVD.cpp

```diff
@@ -17,6 +17,7 @@
 #include "IopCommon.h"
 #include "AppConfig.h"
 
+#include <bitset>
```

Comment:
What was this added for?

Reply:
Oops. When I was printing the binary values of the reg I used cout and bitset. I'll clean that up real quick

Reply:
plz learn to read hex ;)

Reply:
It's been removed :p